R/reportCalibration.R

```diff
@@ -185,27 +208,43 @@ reportCalibration <- function(gdx) {
 #' Compute the deviation to historical data
```

Comment:
Is the function description still accurate?
```suggestion
#' Compute the deviation from target data
```

Reply:
I'd say it's not completely wrong, as the target data is derived from historical data. But I'll change it to be consistent with the rest of the code.